app/src/main/java/org/jellyfin/androidtv/ui/playback/VideoManager.java

```diff
@@ -98,13 +98,18 @@ public VideoManager(PlaybackOverlayActivity activity, View view) {
         mSubtitlesSurface.setZOrderMediaOverlay(true);
         mSubtitlesSurface.getHolder().setFormat(PixelFormat.TRANSLUCENT);
 
-        mExoPlayer = new ExoPlayer.Builder(TvApp.getApplication(), new DefaultRenderersFactory(TvApp.getApplication()) {
+        ExoPlayer.Builder exoPlayerBuilder = new ExoPlayer.Builder(TvApp.getApplication());
```

Comment:
(Pinching the comments from nielsvanvelzen's review on the same part of code)
Both `TvApp.getApplication()` calls should be replaced with `activity`